tests/unit-core/test-api.c

```diff
@@ -630,7 +633,10 @@ main (void)
   TEST_ASSERT (jerry_get_array_length (array_obj_val) == 10);
 
   jerry_value_t v_in = jerry_create_number (10.5);
-  jerry_set_property_by_index (array_obj_val, 5, v_in);
+  res = jerry_set_property_by_index (array_obj_val, 5, v_in);
+  TEST_ASSERT (!jerry_value_is_error (res));
+  TEST_ASSERT (jerry_get_boolean_value (res));
```

Comment:
Please add a `jerry_get_boolean_value` call too to ensure the result was a success. 
```c
TEST_ASSERT (jerry_get_boolean_value (res) && jerry_get_boolean_value (res));
```